prov/sm2/src/sm2_attr.c

```diff
@@ -104,7 +108,7 @@ struct fi_domain_attr sm2_domain_attr = {
 	.data_progress = FI_PROGRESS_MANUAL,
 	.resource_mgmt = FI_RM_ENABLED,
 	.av_type = FI_AV_UNSPEC,
-	.mr_mode = FI_MR_BASIC | FI_MR_SCALABLE,
+	.mr_mode = OFI_MR_BASIC_MAP | FI_MR_LOCAL | FI_MR_BASIC,
```

Comment:
Requesting feedback here to make sure these flags are right

Reply:
It seems the current mode is copied from shm, anything wrong with it? May need to consult @aingerson

Reply:
I switched them back and it sill works.  I don't remember why I changed this in the first place 🤷 .  Removing this change.